languages/elixir/reference/README.md

```diff
@@ -167,4 +167,4 @@ The concept exercises use the following concepts:
 
 | concept | interpretation |
 | --- | --- |
-| `default-arguments` | Extract Elixir specifics from [official guide](https://elixir-lang.org/getting-started/modules-and-functions.html#default-arguments). |
+| `conditionals` | Intro to elixir `cond/1` function and atoms. |
```

Comment:
As corresponding to the `config.json`, there are two Concepts introduced:

```suggestion
| `atoms` | Intro to atoms. |
| `conditionals` | Intro to elixir `cond/1` function. |
```

Reply:
Okay, so separate them, but they can still point to the one exercise. I can do that

Reply:
This document is just an overview of all the concepts that are listed in the `config.json`. As such, there is no direct link to an exercise here. That is done in the `exercises/concept/README.md` file.